Formula/xalan-c.rb

```diff
@@ -18,36 +22,20 @@ class XalanC < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "b1190a346ef882ca4c00f05ea0ccb72b1f148347e6200b6ccfed08613267823c"
   end
 
+  depends_on "cmake" => :build
   depends_on "xerces-c"
 
-  # Fix segfault. See https://issues.apache.org/jira/browse/XALANC-751
-  # Build with char16_t casts.  See https://issues.apache.org/jira/browse/XALANC-773
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/71f4b091e4c4939c3fa95981298580a827788e6f/xalan-c/xerces-char16.patch"
-    sha256 "ebd4ded1f6ee002351e082dee1dcd5887809b94c6263bbe4e8e5599f56774ebf"
-  end
-
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/71f4b091e4c4939c3fa95981298580a827788e6f/xalan-c/locator-system-id.patch"
-    sha256 "7c317c6b99cb5fb44da700e954e6b3e8c5eda07bef667f74a42b0099d038d767"
-  end
-
   def install
     ENV.cxx11
-    ENV.deparallelize # See https://issues.apache.org/jira/browse/XALANC-696
-    ENV["XALANCROOT"] = "#{buildpath}/c"
-    ENV["XALAN_LOCALE_SYSTEM"] = "inmem"
-    ENV["XALAN_LOCALE"] = "en_US"
 
-    cd "c" do
-      system "./configure", "--disable-dependency-tracking",
-                            "--disable-silent-rules",
-                            "--prefix=#{prefix}"
+    mkdir "build" do
```

Comment:
Can we do this instead:
```rb
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DCMAKE_INSTALL_RPATH=#{rpath}"
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
That certainly works but is it preferred to align with the [upstream build instructions](https://apache.github.io/xalan-c/build.html#unix-makefiles-generator-unix) or the approach in similar formulae (the suggestion). If it's the latter, I'll update this accordingly.

Reply:
I know @carlocab has been doing a lot of work to migrate to the suggested form so I'd prefer that option.

Reply:
It's the format you get in `brew create --cmake`, requires fewer changes if you want to change the generator with `-G`, and needs less indentation (which I personally find more readable).

Reply:
Thanks for the clarification. I've added this change in the latest push.